internal/testplanet/identities_table.go

```diff
@@ -6,154 +6,154 @@
-	mustParsePEM("-----BEGIN CERTIFICATE-----\nMIIBQjCB6KADAgECAhEA2vh2ECqXQ5DfEJ92fjbx6DAKBggqhkjOPQQDAjAAMCIY\nDzAwMDEwMTAxMDAwMDAwWhgPMDAwMTAxMDEwMDAwMDBaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAAQ1wuzx8O591DWtVi2Uq3Y4vKY0xUftC5d0bpw19D1mC+Ww\nRK2JC37Tb9zOEmf/i4T3qmHdQOgQUV1ypjptJSOBoz8wPTAOBgNVHQ8BAf8EBAMC\nBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAw\nCgYIKoZIzj0EAwIDSQAwRgIhAOANR76zo36FuHeANfbbZ83QidxtQMRBTpxZJZ3X\n3XMzAiEAqmxggJZmqXbN/ha/fvyxxKlxbhVdf6WzFnLZiXe8ryA=\n-----END CERTIFICATE-----\n-----BEGIN CERTIFICATE-----\nMIIBOTCB4KADAgECAhAfSHOD4pjadUZXO/B9pryfMAoGCCqGSM49BAMCMAAwIhgP\nMDAwMTAxMDEwMDAwMDBaGA8wMDAxMDEwMTAwMDAwMFowADBZMBMGByqGSM49AgEG\nCCqGSM49AwEHA0IABIAFX+aVgZY14RLgsQ+e1rvNZgBg5AW3CvcaTUxrwVVUzMUS\nKQVZn3hSP66qQHtAbfGaVTAHb2XpubqJ1TCTgDCjODA2MA4GA1UdDwEB/wQEAwIC\nBDATBgNVHSUEDDAKBggrBgEFBQcDATAPBgNVHRMBAf8EBTADAQH/MAoGCCqGSM49\nBAMCA0gAMEUCIQCpAdzDN1rlXQ9C1lBeIlunCQHPT9GhyWGaCH9kRALQggIgabY6\n+qgwkwO0bejNB6WOrAuI+5Vmy8u0ireCdIdWwhA=\n-----END CERTIFICATE-----\n", "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIC9Stp1t81ZZz1DZIEYwFLUZVP1jX8ysrVs2/7f2jzckoAoGCCqGSM49\nAwEHoUQDQgAENcLs8fDufdQ1rVYtlKt2OLymNMVH7QuXdG6cNfQ9ZgvlsEStiQt+\n02/czhJn/4uE96ph3UDoEFFdcqY6bSUjgQ==\n-----END EC PRIVATE KEY-----\n"),
-	mustParsePEM("-----BEGIN CERTIFICATE-----\nMIIBQDCB6KADAgECAhEAn/+IKfV/7jrvm7dn92DVyjAKBggqhkjOPQQDAjAAMCIY\nDzAwMDEwMTAxMDAwMDAwWhgPMDAwMTAxMDEwMDAwMDBaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAAQXUpO1RphoaCN5/BYcoUWXg0tHH6nnGHrnPU4KnUGh/ndt\nRwd65xWJ/kUQ4n8Uj6lk1Wwk+VhsmrgLOt40TdeXoz8wPTAOBgNVHQ8BAf8EBAMC\nBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAw\nCgYIKoZIzj0EAwIDRwAwRAIgeRGpzFNxkLJCn2hxVy480UNcoKBN7eKFhijflxpp\ntmgCIBSCVVJ6+Dbr6+Q82ZdrcrAGegEzqmq/I6RNnCYcisM5\n-----END CERTIFICATE-----\n-----BEGIN CERTIFICATE-----\nMIIBOjCB4aADAgECAhEA78KgPBD5FOh7T4KNQOXrnTAKBggqhkjOPQQDAjAAMCIY\nDzAwMDEwMTAxMDAwMDAwWhgPMDAwMTAxMDEwMDAwMDBaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAARkEnmQTEEp4YdyG/e+l5Dj/ZBCTPBAI6PCxfBY+fYEtx5n\nNFpk9kYT6GgbxMAQ7xi7e4NhdLBNHWQVPxPJU1YeozgwNjAOBgNVHQ8BAf8EBAMC\nAgQwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDwYDVR0TAQH/BAUwAwEB/zAKBggqhkjO\nPQQDAgNIADBFAiBCLqJ6xJWz5UDl2qowRxRvTYLi4HG8xlzpX7yVgWtw1AIhAPAM\n7/JGZEeM4D8GWPuVx8OM0CtOkEV/T5xLepZ6KtaD\n-----END CERTIFICATE-----\n", "-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIPnFWk4yAy6vUoDAV06W3ZY+qLOVFVMtLluXtDczhyPwoAoGCCqGSM49\nAwEHoUQDQgAEF1KTtUaYaGgjefwWHKFFl4NLRx+p5xh65z1OCp1Bof53bUcHeucV\nif5FEOJ/FI+pZNVsJPlYbJq4CzreNE3Xlw==\n-----END EC PRIVATE KEY-----\n"),
-	mustParsePEM("-----BEGIN CERTIFICATE-----\nMIIBQDCB6KADAgECAhEAyGn6Pj4LkPqEwleB2oLDcDAKBggqhkjOPQQDAjAAMCIY\nDzAwMDEwMTAxMDAwMDAwWhgPMDAwMTAxMDEwMDAwMDBaMAAwWTATBgcqhkjOPQIB\nBggqhkjOPQMBBwNCAAQrRmS2cCbIGyFUbucCSTcVuOdtvB0XlYqPTIkypVq
```

Comment:
Any reason these need to be changed as well? Since it's generated content, might be a good idea in long-term to avoid changing, due to repository bloat...

Reply:
We need a must parse PEM for the signer which is a certificate authority. I suppose this could remain unchanged, I just thought the name would be too ambiguous at that point.